examples/binaries/waiter/src/lib.rs

```diff
@@ -69,8 +85,8 @@ pub enum RwLockType {
 
 #[derive(Debug, Encode, Decode)]
 pub enum RwLockContinuation {
```

Comment:
What's the purpose of this to be enum?

Reply:
If you are asking about why I have introduced this wrapping enum, then the answer is that I know that I will need it for the tests I have for the mutex ownership, where for the mutex I will have a continuation like `Lock`, whereas for the RW case there will be RW specific continuation like `Lock(RwLockType)`